scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -335,22 +335,20 @@ class FossId(
      */
     private suspend fun getRawResults(scanCode: String): RawResults {
         val responseIdentifiedFiles = service.listIdentifiedFiles(user, apiKey, scanCode)
-            .checkResponse("list identified files", false)
-
+            .checkResponse("list identified files", true)
```

Comment:
commit-message: Please provide a (short) rationale.

Reply:
And please start capitalized after the colon.

Reply:
I try but it's hard to justify more than "it's better" :)